Scripts/parameters.py

```diff
@@ -1095,71 +1095,71 @@
-                "cost": (-0.411197516348e-1, -0.976929479166e-1),
+                "time": (-0.0132535252, -0.0180517183),
+                "cost": (-0.1004054942, -0.136755441374),
             },
             "log": {
                 "size": 1.00000000000, # L_S_M
             },
             "size": {
-                "cbd": (numpy.exp(4.82525484469 - 3.0),
-                        numpy.exp(5.46413339352 - 2.0)),
-                "workplaces_own": (numpy.exp(0.326534462277 + 0.1),
-                                   numpy.exp(0.914054873639 + 0.5)),
+                "cbd": (numpy.exp(4.6880073518),
+                        numpy.exp(5.43822704609)),
+                "workplaces_own": (numpy.exp(0.314367891487),
+                                   numpy.exp(0.906947845409)),
                 "workplaces_other": 1,
             },
         },
         "transit": {
             "attraction": {
-                "cbd": (0.483086068108, 0.483086068108),
-                "helsinki_other": (0.2, 0.2),
+                "cbd": (0.491418509053, 0.491418509053),
+                "helsinki_other": (0.00, 0.00),
             },
             "impedance": {
-                "time": -0.111547282384e-1,
-                "cost": ((-0.411197516348e-1) / 60.0, -0.976929479166e-1 / 44.0),
+                "time": -0.0101838292646,
+                "cost": ((-0.100405494227) / 60.0, -0.136755441374 / 44.0),
             },
             "log": {
                 "size": 1.00000000000, # L_S_M
             },
             "size": {
-                "cbd": (numpy.exp(4.82525484469), numpy.exp(5.46413339352)),
-                "workplaces_own": (numpy.exp(0.326534462277), numpy.exp(0.914054873639)),
+                "cbd": (numpy.exp(4.6880073518), numpy.exp(5.43822704609)),
+                "workplaces_own": (numpy.exp(0.314367891487), numpy.exp(0.906947845409)),
                 "workplaces_other": 1,
             },
         },
         "bike": {
             "attraction": {},
             "impedance": {},
             "log": {
-                "dist": (-1.11485765438, -1.35996965070),
+                "dist": (-1.11623384398, -1.3592708114),
                 "size": 1.00000000000, # L_S_M
             },
             "size": {
-                "cbd": (numpy.exp(4.82525484469), numpy.exp(5.46413339352)),
-                "workplaces_own": (numpy.exp(0.326534462277), numpy.exp(0.914054873639)),
+                "cbd": (numpy.exp(4.6880073518), numpy.exp(5.43822704609)),
+                "workplaces_own": (numpy.exp(0.314367891487), numpy.exp(0.906947845409)),
                 "workplaces_other": 1,
             },
         },
         "walk": {
             "attraction": {
-                "own_zone_area_sqrt": (-2.14509122337, -2.07210556201),
+                "own_zone_area_sqrt": (-2.15046600566, -2.08475720417),
             },
             "impedance": {},
             "log": {
-                "dist": -2.66058468415 + 0.1,
```

Comment:
Is some kind of calibration removed?

Reply:
I removed all calibration parameters for calibrated trip purposes. I could not find arguments to return this calibration, so decided to left it out.